cmd/influx_inspect/cardinality/aggregators/aggregators_test.go

```diff
@@ -0,0 +1,319 @@
+package aggregators
+
+import (
+	"bytes"
+	"sync"
+	"testing"
+
+	"github.com/influxdata/influxdb/models"
+	"github.com/stretchr/testify/require"
+)
+
+type result struct {
+	fields uint64
+	tags   uint64
+	series uint64
+}
+
+// Ensure that tags and fields and series which differ only in database, retention policy, or measurement
+// are correctly counted.
+func Test_canonicalize(t *testing.T) {
+	detailed := true
```

Comment:
Can we cover the simple case too? Should be almost the same test with a slightly different checker function.

Reply:
Or do you mean the simpleNode case?  I think it must be the latter.  Easy enough to do.